src/tools/gen-multidom-structured-mesh.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+
+# gen-multidom-structured-mesh.py
+# Write a simple multidomain structured mesh for testing.
+
+# This script requires a conduit installation configured with python3 and hdf5.
+# Make sure PYTHONPATH includes /path/to/conduit/install/python-modules
+
```

Comment:
Could you please list the commands that you ran to generate the meshes that you checked into `axom_data`?